docs-website/src/pages/_components/Hero/index.js

```diff
@@ -33,7 +34,12 @@ const Hero = ({}) => {
               complexity of your data ecosystem.
             </p>
             <p className="hero__subtitle">
-Built with ❤️ by <img src="https://datahubproject.io/img/acryl-logo-light-mark.png" style={{'vertical-align': 'text-top;'}} width="25"/> <a href="https://acryldata.io" target="blank" rel="noopener noreferrer">Acryl Data</a> and <img src="https://datahubproject.io/img/LI-In-Bug.png" width="25"/> LinkedIn.
+              Built with ❤️ by{" "}
+              <img src="https://datahubproject.io/img/acryl-logo-light-mark.png" style={{ "vertical-align": "text-top;" }} width="25" />{" "}
```

Comment:
not urgent, but while we're tweaking this, can we make the logos not protrude so much above the text?

<img width="531" alt="image" src="https://github.com/datahub-project/datahub/assets/12566801/30200ee0-7067-430b-8880-30a1ef8044ff">